src/combinatorics/simple_graph/basic.lean

```diff
@@ -480,6 +504,29 @@ set.to_finset_card _
   (univ : finset G.edge_set).card = G.edge_finset.card :=
 fintype.card_of_subtype G.edge_finset (mem_edge_finset _)
 
+section from_edge_finset
+
+variable (s : finset (sym2 V))
+
+/--
+`from_edge_finset` constructs a `simple_graph` from a finset of edges, without loops.
+-/
+def from_edge_finset : simple_graph V := from_edge_set s
+
+@[simp] lemma from_edge_finset_adj : (from_edge_finset s).adj v w ↔ v ≠ w ∧ ⟦(v, w)⟧ ∈ s := iff.rfl
+
+@[simp] lemma from_edge_finset_edge_finset [fintype G.edge_set] :
+  from_edge_finset (G.edge_finset) = G :=
+by simp only [from_edge_finset, coe_edge_finset, from_edge_set_edge_set]
```

Comment:
When it's been changed to use `from_edge_set` directly, it follows from other simp lemmas, so it is not necessary.
```lean
@[simp] lemma from_edge_finset_edge_finset [fintype G.edge_set] :
  from_edge_set ↑G.edge_finset = G :=
by simp
```

Reply:
Removed the `from_edge_finset` section.